pom.xml

```diff
@@ -9,145 +9,86 @@
+        <maven.surefire.version>2.22.2</maven.surefire.version>
+        <allure.maven.version>2.12.0</allure.maven.version>
+        <allure.version>2.20.1</allure.version>
         <selenium.version>3.141.59</selenium.version>
         <log4j2.version>2.17.1</log4j2.version>
         <slf4j.version>2.0.3</slf4j.version>
+        <testng.version>6.14.3</testng.version>
+        <webdrivermanager.version>5.3.0</webdrivermanager.version>
         <aspectj.version>1.8.13</aspectj.version>
-        <allure-testng-version>1.5.4</allure-testng-version>
-        <allure-maven-version>2.12.0</allure-maven-version>
-        <allure-version>2.20.1</allure-version>
-        <jaxb-api-version>2.3.1</jaxb-api-version>
-        <jaxb-core-version>2.3.0.1</jaxb-core-version>
+        <lombok.version>1.18.20</lombok.version>
     </properties>
 
     <dependencies>
-        <!-- https://mvnrepository.com/artifact/org.seleniumhq.selenium/selenium-java -->
         <dependency>
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-java</artifactId>
             <version>${selenium.version}</version>
         </dependency>
-        <!-- https://mvnrepository.com/artifact/org.seleniumhq.selenium/selenium-support -->
         <dependency>
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-support</artifactId>
             <version>${selenium.version}</version>
         </dependency>
-
-        <!-- https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-core -->
         <dependency>
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-core</artifactId>
             <version>${log4j2.version}</version>
         </dependency>
-        <!-- https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-api -->
         <dependency>
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-api</artifactId>
             <version>${log4j2.version}</version>
         </dependency>
-
-        <!-- https://mvnrepository.com/artifact/org.testng/testng -->
         <dependency>
             <groupId>org.testng</groupId>
             <artifactId>testng</artifactId>
-            <version>6.14.3</version>
+            <version>${testng.version}</version>
         </dependency>
-
-        <!-- https://mvnrepository.com/artifact/org.hamcrest/hamcrest -->
-        <dependency>
-            <groupId>org.hamcrest</groupId>
-            <artifactId>hamcrest</artifactId>
-            <version>2.2</version>
-        </dependency>
-
-        <!-- https://mvnrepository.com/artifact/io.github.bonigarcia/webdrivermanager -->
         <dependency>
             <groupId>io.github.bonigarcia</groupId>
             <artifactId>webdrivermanager</artifactId>
-            <version>5.3.0</version>
-        </dependency>
-
-        <!-- https://mvnrepository.com/artifact/org.slf4j/slf4j-api -->
-        <dependency>
```

Comment:
Hi anh Khiem, Do you really want to remove the "slf4j" library?

Reply:
@tuanna0308 do we use this lib in the project? if yes, I will revert this